djvm/src/main/kotlin/net/corda/djvm/source/SourceClassLoader.kt

```diff
@@ -42,25 +42,15 @@ class BootstrapClassLoader(
  * @property classResolver The resolver to use to derive the original name of a requested class.
  * @property bootstrap The [BootstrapClassLoader] containing the Java APIs for the sandbox.
  */
-class SourceClassLoader private constructor(
+class SourceClassLoader(
     paths: List<Path>,
     private val classResolver: ClassResolver,
-    private val bootstrap: BootstrapClassLoader?,
-    parent: ClassLoader?
-) : URLClassLoader(resolvePaths(paths), parent) {
+    private val bootstrap: BootstrapClassLoader? = null
```

Comment:
Why has boostrapclassloader gained a default value?

Reply:
It hasn't really. I have removed both of this class's secondary constructors to leave just the primary constructor, and so the primary constructor has "inherited" `bootstrapClassLoader`'s default value.